src/vivarium_cluster_tools/psimulate/cli.py

```diff
@@ -83,8 +85,8 @@ def run(model_specification, branch_configuration, result_directory, **options):
     utilities.configure_master_process_logging_to_terminal(options['verbose'])
     main = handle_exceptions(runner.main, logger, options['with_debugger'])
 
-    main(model_specification, branch_configuration, result_directory,
-         options['project'], options['peak_memory'], options['max_runtime'],
+    native_specification = runner.NativeSpecification(**options, job_name=Path(model_specification).name)
```

Comment:
Do this unpacking inside main.  You're throwing an error that the exception handler can't catch.